cristin-commons/src/test/java/no/unit/nva/utils/UserUtilsTest.java

```diff
@@ -0,0 +1,26 @@
+package no.unit.nva.utils;
+
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.net.URI;
+import java.util.Set;
+
+import static no.unit.nva.testutils.RandomDataGenerator.randomString;
+
+@Tag("integrationTest")
+class UserUtilsTest {
+
+    @Test
+    void createUserWithRoles() throws IOException, InterruptedException {
+
+        String username = "ninja@test";
+        String password = randomString();
+        String nin = "27034425812";
```

Comment:
dev…

Reply:
It's a integration test for manually checking if environment is correct set up,  will not not as part of deployment